#!/usr/bin/env python
PACKAGE = "ros_translator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('translator', str_t, 0, 'Backend Translator Service', 'google')
gen.add('from_language', str_t, 0, 'Input language', 'auto')
gen.add('to_language', str_t, 0, 'Output Language', 'en')
gen.add('api_key', str_t, 0, 'api_key for translator', '')

exit(gen.generate(PACKAGE, 'ros_translator', 'ROSTranslator'))
